ziti/src/main/kotlin/org/openziti/impl/ZitiContextImpl.kt

```diff
@@ -189,6 +190,30 @@ internal class ZitiContextImpl(internal val id: Identity, enabled: Boolean) : Zi
                 throw it
             }
 
+            val mfa = apiSession?.authQueries?.find { it.typeId == MFAType.MFA && it.provider == "ziti" }
```

Comment:
In the future, it is planned that `authQueries` will have other posture checks in them that need to be fed to other posture request handlers. It is possible that any posture check/query type can be in the `authQueries` property. It can't right now however as we are missing the Ziti Controller capability to do so ("authentication policies")